src/callbacks_stage/positivity_shallow_water.jl

```diff
@@ -0,0 +1,85 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+ # Since these FMAs can increase the performance of many numerical algorithms,
+ # we need to opt-in explicitly.
+ # See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+ @muladd begin
+
+
+  """
+      PositivityPreservingLimiterShallowWater(; threshold, variables)
+ 
+  This fully-discrete positivity-preserving limiter is based on the work of 
+  - Zhang, Shu (2011)
+    Maximum-principle-satisfying and positivity-preserving high-order schemes
+    for conservation laws: survey and new developments
+    [doi: 10.1098/rspa.2011.0153](https://doi.org/10.1098/rspa.2011.0153)
+ 
+  It is specifically designed for `ShallowWaterEquations`.
+  The limiter is applied to all scalar `variables` in their given order
+  using the associated `thresholds` to determine the minimal acceptable values.
+  The order of the `variables` is important and might have a strong influence
+  on the robustness. 
+  As opposed to the standard version of the [PositivityPreservingLimiterZhangShu](@ref), 
```

Comment:
```suggestion
  As opposed to the standard version of the [`PositivityPreservingLimiterZhangShu`](@ref), 
```
I think this will fix the failure in the docs build.